File: simpleClass/python3/start.py
from src.UserInfo import UserInfo
import time

antes = time.time() * 1000
list = []

for index in range(1000000):
    user = "user{}".format(index)
    password = "password{}".format(index)
    list.append(UserInfo(user, password))

depois = time.time() * 1000

print(depois - antes)

File: simpleClass/ruby/start.rb
require './src/UserInfo.rb'

antes = Time.now

list = Array.new
(0..1000000).each do |index|
    user = "user#{index}"
    password = "password#{index}"
    list.push(UserInfo.new(user, password))
end

depois = Time.now

p (depois - antes)* 1000.0